controllers/membersController.ts

```diff
@@ -1,27 +1,47 @@
 import { Request, Response } from "express";
-import { Members, NpoMembers } from "../db/models";
+import { Members, NpoMembers, Roles } from "../db/models";
 
-interface MembersAttributes {
+interface NpoMember {
+  id: number;
+  npo_id: number;
+  member_id: number;
+  role_id: number;
+  role?: string;
+  open_ended_ans_1: string | null;
+  open_ended_ans_2: string | null;
+  open_ended_ans_3: string | null;
+  createdAt: string;
+  updatedAt: string;
+}
+
+export interface Member {
   id: number;
   full_name: string;
-  date_of_birth: Date;
-  gender: string;
+  date_of_birth: string;
+  gender: "male" | "female";
   occupation: string;
-  employee_at?: string | undefined;
+  employee_at: string;
   email: string;
-  cv_url?: string | undefined;
-  portfolio_link_url?: string | undefined;
+  cv_url: string;
```

Comment:
I think we might have accidentally removed the nullable option here?

Reply:
Fixed!